addons/mass_mailing/models/mailing_contact.py

```diff
@@ -136,6 +138,8 @@ def create(self, vals_list):
 
         if default_list_ids:
             for vals in vals_list:
+                if vals.get('list_ids'):
```

Comment:
Won't we have issues if we give both list_ids and subscription_list_ids at create ? I think we have to use one or the other, but if you use the two result is not deterministic. I had issues with that kind of behavior with crm.team and team_members (o2m and m2m).

Reply:
Currently subscription_list_ids is supported and convert "default_list_ids". We could also convert "list_ids" as well and work only with ``subscription_list_ids`` maybe ?

Reply:
@tde-banana-odoo Hum, if we want to be robust, that code will be uggly, as you can either give x2many command, or just a list of integer... It means that we have to parse those command and we can technically have weird situation (e.g. updating a list values with list_ids and removing the same list with a x2many command in subscription_list_ids)...

What about making a computed / inverse field with `list_ids` ?